project/src/main/webapp/index.html

```diff
@@ -6,78 +6,88 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
-    <div class="section">
-      <div id="hamburger-icon" onclick="toggleNavBar()"><div></div></div>     
-      <nav id="nav" style="display: none">
-        <ul>
-          <li><a href="#">home</a></li>
-          <li><a href="#trend-frequency">trends</a></li>
-          <li><a href="#trending-topics">topics</a></li>
-        </ul>
-      </nav>
-      <div id="main">
-        <div id="title">trending topics.</div>
-        <a href="#trend-frequency" id="arrow-transition"></a>
+  <body onload="showPreloader()">
+    <div id="preloader">
+      <div id="preloader-container">
+        <div class="down"></div>
+        <div class="up"></div>
+        <div class="down"></div>
+        <div class='up'></div>
       </div>
     </div>
-    <div class="section" id="trend-frequency">
-      <div id="trend-frequency-data">
-        <div>Covid-19 Symptoms: 100,000</div>
-        <div>Mitch McConnell: 100,000</div>
-        <div>Hand Sanitizer: 50,000</div>
-        <div>Minecraft Nether Update: 50,000</div> 
+    <div id="content">
+      <div class="section">
+        <div id="hamburger-icon" onclick="toggleNavBar()"><div></div></div>     
```

Comment:
nit: putting the divs on separate lines so it's easy to visualize which divs are closed and which remain open e.g
<div content>
  <div section>
     <div ham>
     </div>
  </div>
</div>